src/renderer/components/reader/ReaderHeader.tsx

```diff
@@ -75,6 +77,11 @@ export class ReaderHeader extends React.Component<Props, undefined> {
                 this.enableFullscreenRef.current.focus();
             }
         }
+
+        if (this.props.infoOpen !== oldProps.infoOpen &&
+            this.props.infoOpen === false) {
+                this.infoMenuButtonRef.current.focus();
```

Comment:
I agree with this logic, but I just want to point out that when we will implement keyboard shortcuts so that users can quickly invoke main features - like the book info panel, for example with `CTRL i` - then the keyboard focus will not necessarily have been on the info button (the popup dialog was triggered by a combination of keystrokes whilst the keyboard focus was somewhere else in the UI).
Anyway, this is a global problem we will have to solve if we decide to *not* steal the focus when invoking keyboard shortcuts.
Right now, let's keep it simple.